ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlViewModel.cs

```diff
@@ -0,0 +1,90 @@
+﻿
+using ChustaSoft.Common.Base;
+using ChustaSoft.Common.Helpers;
+using ChustaSoft.Common.Models;
+using ChustaSoft.GamersPlatformUtils.Abstractions;
+using ChustaSoft.GamersPlatformUtils.Services;
+using ChustaSoft.GamersPlatformUtils.UI.Helpers;
+using ChustaSoft.GamersPlatformUtils.UI.Styles;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace ChustaSoft.GamersPlatformUtils.UI.Modules.Linker
+{
+    public class LinkerControlViewModel : TraceableViewModelBase<LinkerControlModel>
+    {
+        private const string STEAM_PLATFORM_NAME = "steam";
```

Comment:
Use Steam public constant in it's own layer instead

Reply:
Fixed